setup.py

```diff
@@ -33,10 +33,10 @@
     maintainer_email="luisza14@gmail.com",
     url="http://github.com/xhtml2pdf/xhtml2pdf",
     keywords="PDF, HTML, XHTML, XML, CSS",
-    install_requires=["html5lib>=1.0.1", "pypdf>=3.0.0", "Pillow>=8.1.1",
+    install_requires=["html5lib>=1.0.1", "pypdf>=3.1.0", "Pillow>=8.1.1",
                       "reportlab>=3.5.53", "svglib>=1.2.1",
                       "python-bidi>=0.4.2", "arabic-reshaper>=3.0.0",
-                      "pyHanko>=0.12.1", "pypdf>=3.1.0",
+                      "pyHanko>=0.12.1",
                       "pyhanko-certvalidator>=0.19.5"],
```

Comment:
```suggestion
                      "pyHanko>=0.12.1", "pyhanko-certvalidator>=0.19.5"],
```
What do you think about combine these in one line?
Or write each dependency in exactly one line?
```
    install_requires=[
        "html5lib>=1.0.1",
        "pypdf>=3.1.0",
        "Pillow>=8.1.1",
        "reportlab>=3.5.53",
        "svglib>=1.2.1",
        "python-bidi>=0.4.2",
        "arabic-reshaper>=3.0.0",
        "pyHanko>=0.12.1",
        "pyhanko-certvalidator>=0.19.5",
    ],
```

Reply:
Maybe even consider sorting them alphabetically to avoid such mistakes in the future?